documentation/8-errors.md

```diff
@@ -3,9 +3,9 @@
 ## Errors
 
 Source code:
-- [`source/core/errors.ts`](source/core/errors.ts)
-- [`source/as-promise/types.ts`](source/as-promise/types.ts)
-- [`source/core/response.ts`](source/core/response.ts)
+- [`../source/core/errors.ts`](source/core/errors.ts)
+- [`../source/as-promise/types.ts`](source/as-promise/types.ts)
+- [`../source/core/response.ts`](source/core/response.ts)
```

Comment:
I think you meant to update the actual URL and not the title.

Reply:
fixed 🤦‍♂️